src/views/Home.vue

```diff
@@ -1,34 +1,32 @@
 <template>
-<div class="img" v-bind:style="{ backgroundImage: 'url(' + image + ')' }">
-    <my-banner />
-<b-container>
-<div class="row" style="padding-top: 5%;padding-bottom: 5%;">
-    <div class="col-md-4">
-    <my-grid/>
-    </div> 
-    <div class="col-md-4">
-    <my-grid/>	
-    </div> 
-    <div class="col-md-4">
-        <my-grid/>
-    </div> 
-</div> 
+    <div class="img" v-bind:style="{ backgroundImage: 'url(' + image + ')' }">
+        <my-banner />
+        <b-container>
+            <div class="row" style="padding-top: 5%;padding-bottom: 5%;">
+                <div class="col-md-4">
+                    <my-grid/>
+                </div> 
+                <div class="col-md-4">
+                    <my-grid/>	
+                </div> 
+                <div class="col-md-4">
+                    <my-grid/>
+                </div> 
+            </div> 
 
-<div class="row" style="padding-top: 5%;padding-bottom: 5%;">
-    <div class="col-md-4">
-        <my-grid/>
-    </div> 
-    <div class="col-md-4">
-        <my-grid/>
-    </div> 
-    <div class="col-md-4">
-    <my-grid/>
+            <div class="row" style="padding-top: 5%;padding-bottom: 5%;">
```

Comment:
style="padding-top: 5%;padding-bottom: 5%;" should be improved. In inline should only contain maximum 1 property.